samples/cpp/benchmark_app/utils.cpp

```diff
@@ -767,6 +768,145 @@ void load_config(const std::string& filename, std::map<std::string, ov::AnyMap>&
 }
 #endif
 
+void dump_property(const std::string& filename, const std::map<std::string, ov::AnyMap>& config) {
```

Comment:
void dump_property(const std::string& filename, const std::map<std::string, ov::AnyMap>& property) ?

Reply:
I will rename it.